contrib/dracut/systemd/zfs-dracut-bootfs.service.in

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Set BOOTFS environment for dracut
+Documentation=man:zfs(8)
+DefaultDependencies=no
+After=zfs-import.target
+Requires=zfs-import.target
+
+[Service]
+Type=oneshot
+ExecStart=/bin/sh -c "/bin/systemctl set-environment BOOTFS=$(@sbindir@/zpool list -H -o bootfs | grep -v '^-$')"
```

Comment:
I believe you should add `-m1` to the grep so it exits on first match (like the previous awk).